regression/pages/common/tests/test_consts.py

```diff
@@ -0,0 +1,311 @@
+        PZBEQDSAVH26PPH4,https://payments.mitprofessionalx-stage.mit.edu/
+        coupons/offer/?code=PZBEQDSAVH26PPH4
+        DNMV2T6PJAUVZUHC,https://payments.mitprofessionalx-stage.mit.edu/
+        coupons/offer/?code=DNMV2T6PJAUVZUHC
+        """
+    },
+    'invalid_data': {
+        'source_data': """
+        Order Number:,MITPE-116375
+
+        Enrollment code for professional seat in Internet of Things: Roadmap
+        to a Connected World
+        """,
+        'result': "Coupons not found"
+    }
+}
+
+REMOVE_SPACES_FROM_LIST = {
+    'list_with_spaces': {
+        "original_list": [
+            'string with inner spaces',
+            ' leftrightspaces ',
+            ' outer and inner 12354 '
+        ],
+        "resulting_list": [
+            'stringwithinnerspaces',
+            'leftrightspaces',
+            'outerandinner12354'
+        ]
+    },
+    'list_without_spaces': {
+        "original_list": [
+            'nospaces',
+            'charsand12354',
+            ' ',
+        ],
+        "resulting_list": [
+            'nospaces',
+            'charsand12354',
+            ''
+        ]
+    },
+    'empty_list': {
+        "original_list": [],
+        "resulting_list": []
+    }
+}
+
+EXTRACT_MMM_DD_YYYY = {
+    'course_start_date': {
+        'original_string': 'Course starts: May 04, 2016',
+        'result': 'May 04, 2016'
+    },
+    'discount_end_date': {
+        'original_string': 'Discount valid until Oct 31, 2016',
+        'result': 'Oct 31, 2016'
+    },
+    'invalid_format': {
+        'original_string': 'Discount valid until 31-OCT-1994',
+        'result': 'Required date pattern not found in search string'
+    }
+}
+
+CONVERT_DATE = {
+    'first': {
+        'original_date': 'Oct 31, 2016',
+        'original_format': '%b %d, %Y',
+        'required_format': '%Y-%m-%dT%H:%M:%S',
+        'result': '2016-10-31T00:00:00'
+    },
+    'second': {
+        'original_date': '2016-10-31T00:00:00',
+        'original_format': '%Y-%m-%dT%H:%M:%S',
+        'required_format': '%b %d, %Y',
+        'result': 'Oct 31, 2016'
+    },
+    'third': {
+        'original_date': '2013-1-25',
+        'original_format': '%Y-%m-%d',
+        'required_format': '%m/%d/%y',
+        'result': '01/25/13'
+    },
+    'fourth': {
+        'original_date': '15/2/2010',
+        'original_format': '%d/%m/%Y',
+        'required_format': '%a %b %d %Y',
+        'result': 'Mon Feb 15 2010'
+    },
+    'invalid_date': {
+        'original_date': 'ABC-1-25',
+        'original_format': '%d/%m/%Y',
+        'required_format': '%a %b %d %Y',
+        'result': 'Invalid date or format'
+    },
+    'invalid_format': {
+        'original_date': '2013-1-25',
+        'original_format': 'abcdef',
+        'required_format': '%a %b %d %Y',
+        'result': 'Invalid date or format'
+    }
+}
+
+EXTRACT_NUMERICAL_VALUE = {
+    'normal_string_with_dollar_sign': {
+        "price_strings": '$999',
```

Comment:
You want other strings here. I see one further below, but there's also `$9999.99`, `$.99`, `$0`, `$99,999`.